app/presenters/tree_node/miq_product_feature.rb

```diff
@@ -0,0 +1,54 @@
+module TreeNode
+  # Current assumption is that this builder would only be used in Rbac Features tree
+  #
+  class MiqProductFeature < TreeNode::Menu::Node
+    set_attribute(:key)     { "#{@options[:node_id_prefix]}__#{@object.identifier}" }
+    set_attribute(:title)   { _(@object.name) }
+    set_attribute(:tooltip) { _(@object.description) || _(@object.name) }
+
+    set_attribute(:icon) do
+      case @object.feature_type
+      when 'view'     then 'fa fa-search'
+      when 'control'  then 'fa fa-shield'
+      when 'admin'    then 'pficon pficon-edit'
+      else
```

Comment:
This doesn't look very good with having the `else` in a separate row...use the multiline syntax without the `then` instead.